app/scripts/modules/dcos/job/general.component.html

```diff
@@ -0,0 +1,54 @@
+<div class="container-fluid form-horizontal">
+  <div class="form-group">
+    <div class="col-md-3 sm-label-right">
+      ID
+      <!--<help-field key="dcos.serverGroup.stack"></help-field>-->
+    </div>
+    <div class="col-md-7"><input type="text" class="form-control input-sm" name="id" ng-model="$ctrl.general.id"/></div>
+  </div>
+  <!--<div class="form-group row slide-in" ng-if="basicSettings.region.$error.pattern">-->
+    <!--<div class="col-sm-9 col-sm-offset-2 error-message">-->
+      <!--<span>ID can only contain lowercase letters, numbers, dashes(-).</span>-->
+    <!--</div>-->
+  <!--</div>-->
+  <div class="form-group">
+    <div class="col-md-3 sm-label-right">
+      Description
+    </div>
+    <div class="col-md-7"><input type="text" class="form-control input-sm" name="description" ng-model="$ctrl.general.description"/></div>
+  </div>
+  <div class="form-group">
+    <div class="col-md-3 sm-label-right">
+      CPUs
+    </div>
+    <div class="col-md-2">
+      <input type="number" class="form-control input-sm" name="cpus" ng-model="$ctrl.general.cpus" min="0.01" required/>
```

Comment:
So the image is showing as required because of the dockerImageAndTagSelector component, there is no way to directly control that without editing the component itself.